subtests/docker_cli/systemd/systemd_build.py

```diff
@@ -0,0 +1,28 @@
+"""
```

Comment:
Yes, the convention (by other tests) is to just bundle the small ones in the main module file.